Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/.template.config/template.json

```diff
@@ -32,6 +32,10 @@
                 "description": "Target .NET 7"
                 },
                 {
+                "choice": "net8.0",
```

Comment:
I think this is the only file I need to update for .net 8 support in this repo. Once merged and we release a nuget package which include this change, will use that in the cli feed to continue my testing.